components/home/integrationCard.tsx

```diff
@@ -0,0 +1,88 @@
+import { Card, Badge } from "react-bootstrap";
+import SmallIntegration from "./smallIntegration";
+import Link from "next/link";
+import { formatNameIntoLink } from "../../utils/formatters";
+
+export default function IntegrationCard({
```

Comment:
There's something wonky about the corners of the card
<img width="56" alt="Screen Shot 2020-10-09 at 3 14 46 PM" src="https://user-images.githubusercontent.com/303226/95635786-42d05180-0a42-11eb-9ad7-d0f46783f3e2.png">